extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -114,6 +121,9 @@ export class LightboxManager {
       return this.initPromise_;
     }
     this.initPromise_ = this.scanLightboxables_();
```

Comment:
I think you need to call `maybeInit()` instead of `this.scanLightboxables_`, but change the behavior of `maybeInit()` to rescan all the time instead of only when `this.initPromise_` is not initialized. Basically, you need to set `this.initPromise_` to the new promise returned by `this.scanLightboxables`, because we depend on this promise in the function `getElementsForLightboxGroup(lightboxGroupId)`.